src/lib/gravity/index.ts

```diff
@@ -6,10 +6,30 @@ class Gravity {
     production: "api.artsy.net",
   }
 
+  async getAccessToken(credentials: Credentials) {
+    const gravityUrl = this.url("oauth2/access_token")
+    const body: AccessTokenRequest = {
+      client_id: process.env.CLIENT_ID!,
+      client_secret: process.env.CLIENT_SECRET!,
+      grant_type: "credentials",
+      ...credentials,
+    }
+
+    const response = await fetch(gravityUrl, {
+      method: "post",
+      body: JSON.stringify(body),
+      headers: { "Content-Type": "application/json" },
+    })
+
+    const json = await response.json()
+
+    return json as AccessTokenResponse
+  }
+
   async get(endpoint: string) {
     const token: string = process.env.TOKEN! // temp until our auth/token plumbing is hooked up
 
-    const gravityUrl: string = this.url(endpoint)
+    const gravityUrl: string = this.url(`api/v1/${endpoint}`)
```

Comment:
I had to do a little bit of shuffling around of URLs, because the auth endpoint isn't under `api/v1...`.